src/main/resources/layouts/report.fxml

```diff
@@ -77,6 +78,16 @@
                               </Label>
                            </children>
                         </HBox>
+                        <HBox prefHeight="26.0" prefWidth="210.0">
+                           <children>
+                              <Label prefHeight="26.0" prefWidth="137.0" text="Collapse/Expand: ">
```

Comment:
not sure if we need a label "Collapse/Expand" when the button has the same info "Collapse"?

Reply:
Removed the unnecessary label